tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -23,9 +23,16 @@ module.exports = {
       return util.format(this.elements.permissionButton.selector, permission)
     },
 
-    assertSharingNotAllowed: function () {
+    assertSharingNotAllowed: function (resource) {
       // eslint-disable-next-line no-unused-expressions
-      this.api.expect.element(this.elements.sharingAutoComplete.selector).not.to.be.present
+      this.api.page
+        .FilesPageElement
+        .filesList()
+        .closeSidebar(100)
+        .openSharingDialog(resource)
```

Comment:
Does this not introduce a coupling between two different page objects? Is opening/closing sidebar the concern of sharing dialog? I'd suggest we move this outside of the page objects, but, please comment if you feel otherwise.